sdks/python/apache_beam/transforms/__init__.py

```diff
@@ -24,3 +24,4 @@
 from apache_beam.transforms.ptransform import *
 from apache_beam.transforms.timeutil import *
 from apache_beam.transforms.util import *
+from apache_beam.transforms.display import *
```

Comment:
Do we really want to import everything into the global(ish) apache_beam namespace? The typical user will just be returning dicts, and at most DisplayDataItems (which I think are fair game to import manually).